frontend/src/modules/history/components/Translation.js

```diff
@@ -138,6 +199,36 @@ export default class Translation extends React.Component<Props> {
                         />
                     </div>
                     <menu className='toolbar'>
+                    { index === 0 ? null :
+                        this.state.isDiffVisible ?
```

Comment:
I think double ternary operators is not a great pattern, it really hinders readability. When that happens in my code, I tend to extract that code to a `renderX` method, and use regular `if` to return the expected result.